sycl/test-e2e/Basic/vector/vec_binary_scalar_order.cpp

```diff
@@ -0,0 +1,151 @@
+// REQUIRES: preview-breaking-changes-supported
+// RUN: %{build} -fpreview-breaking-changes -o %t.out
+// RUN: %{run} %t.out
+
+// This test currently fails on AMD HIP due to an unresolved memcmp function.
+// XFAIL: hip_amd
```

Comment:
@npmiller - It seems that this test fails on AMD HIP due to `memcmp`. I suspect it might be the vector comparison operators, but I have not reproduced it locally. Once this is merged I will open an issue for it.

Reply:
Is that a stdlib memcmp? We haven't added libcxx support for AMD yet so that would make sense.

Reply:
The error was:
```
# .---command stderr------------
# | lld: error: undefined hidden symbol: memcmp
# | >>> referenced by lto.tmp:(typeinfo name for main::'lambda163'(sycl::_V1::handler&)::operator()(sycl::_V1::handler&) const::'lambda'())
# | >>> referenced by lto.tmp:(typeinfo name for main::'lambda163'(sycl::_V1::handler&)::operator()(sycl::_V1::handler&) const::'lambda'())
# | >>> referenced by lto.tmp:(typeinfo name for main::'lambda263'(sycl::_V1::handler&)::operator()(sycl::_V1::handler&) const::'lambda'())
# | >>> referenced 1 more times
# | llvm-foreach: 
# | clang++: error: amdgcn-link command failed with exit code 1 (use -v to see invocation)
# `-----------------------------
```

Reply:
Yeah that sounds like it, it's fine to keep it as XFAIL